Tests/test_Nexus.py

```diff
@@ -13,7 +13,6 @@
 import sys
 from Bio._py3k import StringIO
 from Bio._py3k import range
-from Bio.AlignIO import NexusIO
```

Comment:
Based on the changes below, you probably meant to replace this with:

``` python
from Bio.AlignIO.NexusIO import NexusIterator
```

This explains at least some of the unit-test failures reported by TravisCI.